posts/2022-08-30-password-utilities-22009.adoc

```diff
@@ -0,0 +1,160 @@
++
+The issue is fixed via link:https://github.com/OpenLiberty/open-liberty/pull/21883[#21883] and 
+Open Liberty will now reject LTPA tokens when the two expiration times in the token are not within an acceptable difference by throwing an invalidTokenException.
+
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/21955[Liberty does not provide exported packages for java.* packages at runtime in the OSGi framework instance]
++
+Starting with OSGi R7 Core specification, bundles are allowed to import (using `Import-Package` header) packages from `java.*` (e.g. `java.io`).  When installing a liberty feature that includes a bundle that imports a `java.*` package the bundle fails to resolve because Liberty does not configure the OSGi framework (Equinox in this case) to export the `java.*` packages available in the running JVM.
++
+The issue is resolved via link:https://github.com/OpenLiberty/open-liberty/pull/16188[#16188] and Open Liberty will now configure the OSGi framework to export the available java.* packages for the running JVM.
+
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/21937[MP Fault Tolerance 1.x can log an FFDC when a method times out at the same time as it completes]
++
+When a method times out at the same time as it completes, an FFDC may be logged. Aside from the FFDC being logged, there are no other symptoms.
++
+There has been a fix to timeout cancellation race condition which resolves:
++
+A lock shared between `stop()` and `timeout()`, but if the timeout
+has fired, the timeout future is not complete at the point where
+`timeout()` releases the lock. This gives `stop()` a chance to run and
+attempt to cancel the future. Doing this results in an
+`InterruptedException` which causes and FFDC to be emitted by the policy
+executor.
++
+By checking the value of timeout, it avoids trying to cancel the
+timeout task if the timeout has already fired and interrupted our
+thread. There has also been a package added to the FT trace group.
++ 
+Both the fix to the timeout cancellation race condition and adding the package to trace group has resolved the issue of link:https://github.com/OpenLiberty/open-liberty/pull/21936[MP Fault Tolerance 1.x]. 
+
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/21880[OpenAPI 2.0+ throws error at startup]
++
+After enabling mpOpenApi-2.0 or mpOpenApi-3.0. at startup, an error which can be found 
+link:https://github.com/OpenLiberty/open-liberty/issues/21880[here] may occur because the MP OpenAPI implementation tries to get a MicroProfile Config object before the MicroProfile Config feature has initialized.
++
+This issue was fixed by only calling `Config.getConfig()` when needed, rather than when
+the component starts to avoid occasional cases where the config component hasn't initialized yet. The fixes around found in link:https://github.com/OpenLiberty/open-liberty/pull/21881[#21881]
```

Comment:
```suggestion
the component starts to avoid occasional cases where the config component hasn't initialized yet. The fixes are  in link:https://github.com/OpenLiberty/open-liberty/pull/21881[#21881]
```